packages/proxy/src/proxy.ts

```diff
@@ -1522,6 +1521,111 @@ async function fetchAnthropic(
   };
 }
 
+function pruneSchema(schema: any): any {
```

Comment:
If this is specific to google, can you either name it as such or move it into the google-specific file?

Reply:
Done